### PR TITLE
Enable egress github action

### PR DESCRIPTION
**Pull request by robert-bryson**

### CODE REVIEW

File: .github/workflows/enable-egress.yml [hunk 1]
```diff
@@ -11,6 +11,10 @@ on:
         description: 'Cloud Foundry Space'
         required: true
 
+env:
+  DATAGOV_DIR: datagov
```
Comment (nickumia-reisys) [inline .github/workflows/enable-egress.yml:15]: Where is this directory?  I don't see it in main branch nor being added here 😕 Is this specifying the name of the folder that the repo gets cloned to?

**Review by nickumia-reisys — COMMENTED** [review]
Would it be easier to specify the environment as a matrix, so that the same code doesn't have to be duplicated three times? 
Reference: https://docs.github.com/en/actions/using-jobs/using-a-matrix-for-your-jobs#excluding-matrix-configurations

File: .github/workflows/enable-egress.yml (hunk 1 above)
Comment (robert-bryson) [reply .github/workflows/enable-egress.yml:15]: It is used as the path for the [datagov repo](https://github.com/GSA/data.gov/blob/de441e6b2d486410a5e3ce47686e98fb0e38836b/.github/workflows/enable-egress.yml#L30), same as [CG_DIR](https://github.com/GSA/data.gov/blob/de441e6b2d486410a5e3ce47686e98fb0e38836b/.github/workflows/enable-egress.yml#L35), during the execution of the action. They are needed to provide a path so the script is able to reach into the other dir/repo.

**Comment by robert-bryson** [comment]
> Would it be easier to specify the environment as a matrix, so that the same code doesn't have to be duplicated three times? Reference: https://docs.github.com/en/actions/using-jobs/using-a-matrix-for-your-jobs#excluding-matrix-configurations

I believe a matrix in github actions works in the inverse way: provide a matrix of options and run the action with all permutations. We are trying to input a specific option and run an action with just those parameters.

**Review by nickumia-reisys — APPROVED** [review]
⛰️

**Comment by nickumia-reisys** [comment]
Linking to 
- https://github.com/GSA/data.gov/issues/3873

**Pull request merged**